Description: Removes the UserDataHandler interface (already provided by the JDK) to avoid a build failure with Java 9
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/src/java/main/org/w3c/dom/UserDataHandler.java
+++ /dev/null
@@ -1,23 +0,0 @@
-/** This is a W3C interface we include here so that NamespaceNode can compile in both
- *  Java 1.4 and 1.5. It's owned by the W3C, and available under their usual 
- *  extremely liberal license so this shouldn't bother anyone. (XPath itself
- *  is under the same license after all.)
- */
-
-package org.w3c.dom;
-
-public interface UserDataHandler {
-    // OperationType
-    public static final short NODE_CLONED               = 1;
-    public static final short NODE_IMPORTED             = 2;
-    public static final short NODE_DELETED              = 3;
-    public static final short NODE_RENAMED              = 4;
-    public static final short NODE_ADOPTED              = 5;
-
-    public void handle(short operation, 
-                       String key, 
-                       Object data, 
-                       Node src, 
-                       Node dst);
-
-}
\ No newline at end of file
--- a/pom.xml
+++ b/pom.xml
@@ -243,7 +243,6 @@
             <Bundle-Version>${project.version}</Bundle-Version>
             <Export-Package>org.jaxen.*;version=${project.version}</Export-Package>
             <Include-Resource>
-              org/w3c/dom/UserDataHandler.class=target/classes/org/w3c/dom/UserDataHandler.class,
               META-INF/LICENSE.txt=LICENSE.txt
             </Include-Resource>
             <Import-Package>
